src/tools/compiletest/src/runtest.rs

```diff
@@ -571,6 +571,10 @@ impl<'test> TestCx<'test> {
 
         rustc.args(self.split_maybe_args(&self.config.target_rustcflags));
         rustc.args(&self.props.compile_flags);
+        rustc.arg(&format!(
```

Comment:
I would prefer more like

```rust
if let Some(e) = self.props.editions {
  rustc.arg(&format!("--edition={}", e))
}
```

this way we can test the compiler's default behavior